### ISSUE TITLE
Gesture while using in ListView or SingleChildScrollView

### ISSUE THREAD

**Issue by greenlite-devone** [fysoul17/google_maps_place_picker#52]
Map gestures aren't recognized when the PlacePicker widget is a child of ListView or SingleChildScrollView. adding
 ```
gestureRecognizers: Set()
            ..add(Factory<EagerGestureRecognizer>(
                () => EagerGestureRecognizer())),
```
to GoogleMap in GooglePlacePicker solves it

**Comment by faithomotoso** [comment]
Hi can I open a PR for this?

**Comment by fysoul17** [comment]
@faithomotoso PR is always welcome ;)

**Comment by fysoul17** [comment]
@greenlite-devone 
The issue has been solved and can be updated with version 0.10.0
